src/SCRIPTS/BF/radios.lua

```diff
@@ -19,6 +37,24 @@ local supportedRadios =
         textSize        = SMLSIZE,
         yMinLimit       = 12,
         yMaxLimit       = 52,
+        cmsConfig       = {
+            rows = 8,
+            cols = 32,
+            pixelsPerRow = 8,
+            pixelsPerChar = 6,
+            xIndent = 0,
+            yOffset = 0,
+            textSize = SMLSIZE,
+            refresh = {
+                event = EVT_VIRTUAL_INC,
+                text = "Refresh: [+]",
+                top = 1,
+                left = 156,
+            },
+            exit = {
+                event = EVT_VIRTUAL_EXIT
```

Comment:
Seems to be the same everywhere - does this need to be configured per screen size, now that we have virtual events?

Reply:
It would be great if ``cmsConfig`` could be loaded only for the CMS version since it's not needed for the MSP version.

Reply:
can they be called separate versions if they're based on the same framework?

Reply:
@mikeller this virtual event is kind of an assertion, as I don't have all of the hardware to confirm usability in this context.   I'd like to have this tested on multiple platforms to see if it can truly be unified.

Reply:
@klutvott123 would the suggestion then be to have radiosMsp and radiosCms?  I feel like having them in one object is a little more DRY in nature.

Reply:
would it be better to put the MSP related attributes in a nested `msp` table and the cms related attributes into a nested `cms` table (instead of `cmsConfig`)?

Reply:
@codecae I'm using the word "version" loosely 😁. I was thinking maybe pass a variable to radios.lua when executing it and put cms or msp stuff into the ``radio`` table based on that.

Reply:
I added an update to separate the `msp` vs `cms` attributes in the radio config.  I think this is a good compromise.  Let me know what you think.

Reply:
@codecae Looks good to me! Thanks! 👍